class  B
{
	int test()
	{
		try
		{

		}
		catch(ArithmeticException ex)
		{
	
		}
		return 30;
	}
}
